src/components/interface/ConfirmationModal.vue

```diff
@@ -8,23 +8,23 @@
     >
       <div class="p-4 mt-4">
         <div class="mb-4 text-squant">
-          {{ $t("BasePasswordInput.input1.placeholder1") }}
+          {{ $t("ConfirmPassword.headline") }}
         </div>
 
         <PasswordInput show-eye />
       </div>
 
-      <div class="flex flex-col items-center pb-4 mx-4">
+      <div class="pb-4 text-center">
         <Button
           color="green"
-          class="w-full p-4 mt-4"
+          class="w-full md:w-3/4 p-4 m-4 m-auto"
```

Comment:
Alignment/spacing is a bit off before `md` breakpoint

<img width="527" alt="Screen Shot 2021-09-22 at 13 10 36" src="https://user-images.githubusercontent.com/22848332/134414787-ff1c22a3-e752-45cc-aed7-f18c54526fa3.png">

Reply:
Adjusted margins to improve appearance on smaller screens.